src/components/Details/Details.js

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import { ScrollView, Text, ImageBackground, View, Image } from 'react-native';
+import { useTheme } from '@react-navigation/native';
+
+import { PropTypes } from 'prop-types';
+import styles from './styles';
+import MovieInfo from 'components/MovieInfo/MovieInfo';
+import textStyles from 'helpers/TextStyles';
+
+const Details = ({ route }) => {
+  const { colors } = useTheme();
+  const { movie, genres, prefixUrl } = route.params;
+
+  const uri = prefixUrl + movie.backdrop_path;
+  const thumbUri = prefixUrl + movie.poster_path;
```

Comment:
You could destructure the movie in line 12 and get the backdrop_path and poster_path there